packages/static-build/test/gatsby.test.ts

```diff
@@ -281,7 +281,7 @@ describe('gatsby utilities', () => {
     `);
     expect(gatsbyConfig).toMatchInlineSnapshot(`
       "// This file was generated by @vercel/static-build
-      import userConfig from \\"./gatsby-config.mjs.__vercel_builder_backup__.mjs\\";
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏was this also a difference between versions of jest?

Reply:
I'm not 100% certain, but I'm guessing it was a breaking change in v29.0.0: https://github.com/facebook/jest/blob/main/CHANGELOG.md#2900

```
[jest-config] [BREAKING] Make snapshotFormat default to escapeString: false and printBasicPrototype: false (https://github.com/facebook/jest/pull/13036)
```